tests/test_psf.py

```diff
@@ -307,7 +358,7 @@ def checkPiffDeterminer(self, **kwargs):
             chiLim = 7.0
         else:
             numAvail = len(psfCandidateList)
-            chiLim = 6.1
+            chiLim = 6.4
```

Comment:
Why this switched from 6.1 to 6.4?

Reply:
@esheldon increased it to 6.5 and I found 6.4 was sufficient for tests to pass. Do you have an intuition as to why the increase was necessary with the rotations, Erin?

Reply:
I thought it was probably just noise.

Reply:
It could be. I don't think there was any first-principle motivation for `chiLim = 6.1` in the first place, just a value chosen empirically.

Thresholds in `pipelines_check` repo have to be changed only after announcing/asking the entire Science Pipelines team, but these ones can be changed by a reasonable amount.